brainstormforce/ultimate-addons-for-gutenberg
# Fix: Global - Resolved an issue that was encountered when updating to WordPress 6.3 on some websites.

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3252

### Screenshots

- NA

### Types of changes




- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Issue is Reproduce when we are Knight theme, So install it.
- Open a page, Drag and drop Heading block of Spectra.
- Try to transform it to Paragraph, In master you will get an error with white screen.
- Now switch to the branch refresh the page and try the same step, You will not see that error any more.
- Make sure to test Copy/Paste style with Shortcut provided i.e `cntr + shift + y` to copy text and `cntr + shift + u` to paste the style.

### Checklist:
